request.proto

```diff
@@ -104,8 +104,8 @@ message JourneysRequest {
     optional int32 free_radius_from                     = 20 [default=0]; // meters
     optional int32 free_radius_to                       = 21 [default=0]; // meters
     optional int32 min_nb_journeys                      = 22;
-    optional double night_bus_filter_max_factor         = 23;
-    optional int32 night_bus_filter_base_factor         = 24;
+    optional double night_bus_filter_max_factor         = 23 [default=3];
+    optional int32 night_bus_filter_base_factor         = 24 [default=3600]; // seconds
```

Comment:
I'd prefer realistic values as setted in production.

Reply:
Where can I find them ? I took the default values that were already set in Jormungandr.

Reply:
What do you think is a realistic value as different coverages in production have different values ? 
Also, those values are being used all over the place in Jormungandr and the integration tests are based on those specific values (3/3600).

Reply:
You can find them directly in the api: https://api.navitia.io/v1/coverage/fr-idf/status
```
"night_bus_filter_base_factor": 900,
"night_bus_filter_max_factor": 1.5,
```